pkgs/by-name/si/silverbullet/silverbullet

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
I am almost certain that this script can be generated by makeWrapper/wrapProgram.